recipes/autorift/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "autorift" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/leiyangleon/autoRIFT/archive/v{{ version }}.tar.gz
+  sha256: de5656fd5c9476635cc8d26ff0a323da2c8ed92ec2d92b529504f89bafde91fd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - opencv 3.4.*
+  run:
+    - python
+    - opencv 3.4.*
```

Comment:
Do you use the Python bindings? If not this can be removed
```suggestion
```
If you do it's probably better to depend on them directly instead of the meta-package.

Reply:
I actually have to add opencv back to host as well. As we import cv2 to determine the include path to the opencv headers. We also use opencv python bindings in our python scripts.

Reply:
I have also squashed all the commits now